app/src/main/java/com/khinthirisoe/bakingapp/ui/baking/view/BakingViewHolder.kt

```diff
@@ -0,0 +1,28 @@
+package com.khinthirisoe.bakingapp.ui.baking.view
+
+import android.view.View
+import com.khinthirisoe.bakingapp.R
+import com.khinthirisoe.bakingapp.data.model.Recipe
+import com.khinthirisoe.bakingapp.ui.base.BaseViewHolder
+import kotlinx.android.synthetic.main.list_baking.view.*
+
+class BakingViewHolder(itemView: View) : BaseViewHolder(itemView) {
+
+    fun configureUI(baking: Recipe, position: Int) {
+
+        itemView.txt_no_of_serving.text = baking.servings.toString()
+        itemView.txt_baking_name.text = baking.name
+
+        itemView.img_baking.setImageDrawable(itemView.context.resources.getDrawable(getImages(position)))
+
+    }
+
+    fun getImages(position: Int): Int {
+
+        val bakingImages =
+            arrayOf(R.drawable.nutella_pie, R.drawable.brownies, R.drawable.yellow_cake, R.drawable.cheese_cake)
```

Comment:
@KhinThiriSoe you're still creating this array every time you get image id.

Reply:
yes, I've already commented. just need to fix left. I'll think about it. :)

Reply:
still waiting for the fix

Reply:
@KhinThiriSoe 

1. you can use `when` case that will makes it more memory efficient.
2. if you want to have it cleaner, just declare bakingImages as a Companion object and use it directly without the method. This way is not a memory efficient but it's cleaner and more readable.

I prefer 2 because the benefit of readability outweighs the memory usage here